back/src/Helper/BeerHelper.php

```diff
@@ -37,8 +37,8 @@ public function getBeerOrderedList()
 
         $counts = [];
         foreach ($transactions as $transaction) {
```

Comment:
Idéalement, on écrirait une requête SQL pour calculer ce résultat, au lieu de tout charger en mémoire et d'itérer sur les transactions...

Reply:
Oui en effet j'avais pas trop fait attention vu que je changeais peu, je fix ça (très satisfying de remplacer l'énorme boucle par juste une requête)